### PR TITLE
Draw bar from file

### PR DESCRIPTION
**Pull request by ndtquang**
### Card
Finish [#349](https://employmenthero.atlassian.net/jira/software/projects/ANS/boards/93?selectedIssue=ANS-349)
### Decription

- Draw multiple charts that import data from JSON file
- Fix Chart display UI due to Hau's feedback
- Add data folder that contains JSON data

### Screenshot
image

### CODE REVIEW

**Review by haudao — APPROVED** [review]

**Review by haudao — APPROVED** [review]

File: src/components/BarChart.js [hunk 1]
```diff
@@ -3,26 +3,59 @@ import BarGroup from './BarGroup';
...
+    let JSON = require('../data/' + fileName[i]);
+    heroDesign.push({
+      name: JSON.date,
+      value: Math.round(JSON.ratio['value']),
+    });
+    velonic.push({
+      name: JSON.date,
+      value: 100 - Math.round(JSON.ratio['value']),
+    });
+  }
```
[the remark this answers is not in the document]
Comment (ndtquang) [reply src/components/BarChart.js:25]: I have replaced for loop with map in new update, please check

[the remark this answers is not in the document]
Comment (ndtquang) [reply src/components/BarChart.js:25]: I have replaced for-loop with my new update, please check

[the remark this answers is not in the document]
Comment (haudao) [reply src/components/BarChart.js:25]: remove these codes instead of commenting them out

File: src/components/BarChart.js [hunk 2]
```diff
@@ -1,28 +1,74 @@
...
+    fileName => (
+      heroDesign.push({
+        name: require('../data/' + fileName).date,
+        value: Math.round(require('../data/' + fileName).ratio['value']),
+      }),
+      velonic.push({
+        name: require('../data/' + fileName).date,
+        value: 100 - Math.round(require('../data/' + fileName).ratio['value']),
+      })
+    )
```
Comment (haudao) [inline src/components/BarChart.js:38]: have you tested your app again. I don't think this code works, `map` receives. a function but your code is not a right syntax for function

Comment (ndtquang) [reply src/components/BarChart.js:38]: Yes, It does work

**Review by haudao — APPROVED** [review]

**Pull request merged**